vector/src/main/java/im/vector/app/features/login/LoginViewModel.kt

```diff
@@ -417,6 +417,13 @@ class LoginViewModel @AssistedInject constructor(
 
     private fun handleInitWith(action: LoginAction.InitWith) {
         loginConfig = action.loginConfig
+
+        // If there is a pending email validation continue on this step
+        currentThreePid?.let {
```

Comment:
This should be in the init{} of the ViewModel

Reply:
I first tried that but since init is working before the Activity is created, the Activity cannot observe events.